packages/front-end/components/Layout/Layout.tsx

```diff
@@ -211,8 +211,8 @@ const navlinks: SidebarLinkProps[] = [
         name: "Import your data",
         href: "/importing",
         path: /^importing/,
-        filter: ({ permissions, permissionsUtils, gb }) =>
-          permissions.check("manageFeatures", "") &&
+        filter: ({ permissions, permissionsUtils, gb, project }) =>
+          permissionsUtils.canViewFeatureModal(project) &&
```

Comment:
The importer doesn't use the current project at all.  It creates new projects based on the ones in LaunchDarkly and adds features there.  So we really do need to check for global permissions here.